python/cudf_polars/cudf_polars/dsl/expressions/binaryop.py

```diff
@@ -8,13 +8,15 @@
 
 from typing import TYPE_CHECKING, ClassVar
 
-from polars.polars import _expr_nodes as pl_expr
+from polars import polars
 
 import pylibcudf as plc
 
 from cudf_polars.containers import Column
 from cudf_polars.dsl.expressions.base import ExecutionContext, Expr
 
+pl_expr = polars._expr_nodes
```

Comment:
Why this change in the import? AFAICT you just need the same thing below anyway.